cram/cram_decode.c

```diff
@@ -1145,9 +1145,10 @@ static int cram_decode_seq(cram_fd *fd, cram_container *c, cram_slice *s,
 
         if (ncigar+2 >= cigar_alloc) {
             cigar_alloc = cigar_alloc ? cigar_alloc*2 : 1024;
```

Comment:
This false alarm (and the same one in _htslib/kbitset.h_) no longer occurs with current development cppcheck 1.90pre.